performance/Farkle.Benchmarks/Farkle.Benchmarks.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
On my machine Farkle 7 is faster even in .NET 8 but GitHub Actions seems to disagree. Bumping to .NET 9 changed its mind.